idaes/generic_models/unit_models/column_models/column.py

```diff
@@ -0,0 +1,1144 @@
+from idaes.core.util.constants import Constants as CONST
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core import (ControlVolume1DBlock, UnitModelBlockData,
+                        declare_process_block_class,
+                        MaterialBalanceType,
+                        EnergyBalanceType,
+                        MomentumBalanceType,
+                        FlowDirection)
+from idaes.core.util import get_solver
+from idaes.core.util.config import is_physical_parameter_block
+from idaes.core.util.exceptions import ConfigurationError
+from idaes.core.control_volume1d import DistributedVars
+import idaes.logger as idaeslog
+
+
+__author__ = "Paul Akula, John Eslick"
+
+
+# Set up logger
+_log = idaeslog.getLogger(__name__)
+
+
+class ProcessType(Enum):
+    absorber = 1
+    stripper = 2
+
+
+@declare_process_block_class("PackedColumn")
+class PackedColumnData(UnitModelBlockData):
+    """
+    Standard Continous Differential Contactor (CDC) Model Class.
+
+    """
+
+    # Configuration template for unit level arguments applicable to both phases
+    CONFIG = UnitModelBlockData.CONFIG()
+
+    # Configuration template for phase specific  arguments
+    _PhaseCONFIG = ConfigBlock()
+
+    CONFIG.declare("finite_elements", ConfigValue(
+        default=10,
+        domain=int,
+        description="Number of finite elements length domain",
+        doc="""Number of finite elements to use when discretizing length
+domain (default=20)"""))
+
+    CONFIG.declare("length_domain_set", ConfigValue(
+        default=[0.0, 1.0],
+        domain=list,
+        description="List of points in length domain",
+        doc="""length_domain_set - (optional) list of point to use to
+initialize a new ContinuousSet if length_domain is not
+provided (default = [0.0, 1.0])"""))
+
+    CONFIG.declare("transformation_method", ConfigValue(
+        default="dae.finite_difference",
+        description="Method to use for DAE transformation",
+        doc="""Method to use to transform domain. Must be a method recognised
+by the Pyomo TransformationFactory,
+**default** - "dae.finite_difference".
+**Valid values:** {
+**"dae.finite_difference"** - Use a finite difference transformation method,
+**"dae.collocation"** - use a collocation transformation method}"""))
+
+    CONFIG.declare("collocation_points", ConfigValue(
+        default=3,
+        domain=int,
+        description="Number of collocation points per finite element",
+        doc="""Number of collocation points to use per finite element when
+discretizing length domain (default=3)"""))
+
+    CONFIG.declare("column_pressure_drop", ConfigValue(
+        default=0,
+        description="Column pressure drop per unit length in Pa/m",
+        doc="Column pressure drop per unit length in Pa/m provided as a value or expression"))
+
+    # Populate the phase side template to default values
+    _PhaseCONFIG.declare("has_pressure_change", ConfigValue(
```

Comment:
This should be a global level argument - this setting should be the same for both phases.